### PR TITLE
design: implement banner campaign

### LINKED ISSUE #444: remove separator in injected content (e.g. videos)
The separator in videos is also included when injected. This might also occur on other content...

bildschirmfoto vom 2016-10-20 12 50 32

### LINKED ISSUE #515: Implement banner campaign
The general idea is to use images and text below the usual content to create brand awareness and show the people working behind the scenes.

I'd strongly prefere a coded implementation.

**Open Questions**
- [ ] How to store the content? (static or db)
- [ ] Show on every page / always or just once every few pages

**Position**
Between content and comments.

**Layout**
bildschirmfoto 2017-05-09 um 16 20 12

**Implementation thoughts**
- div with green area and photo as background
- height determined by _min height_ and responding to text length

**Small screens version**
smth. like this
image

### PR DESCRIPTION
**Pull request by Knorrke**
This pull request includes:
- New Banner design on top of horizon implementation. (closes #515)
- Refactoring of discussions (needed for placing horizon between content and discussions. Also closes #444)

### CODE REVIEW

**Comment by Knorrke** [comment]
Screenshot on page without discussions:
grafik

Screenshot with discussions:
grafik

**Comment by Knorrke** [comment]
No, we allready did this one or two months ago 😅

File: src/module/Ui/templates/taxonomy/term/templates/curriculum-topic-folder.twig [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {% set typename = 'curriculum-topic-folder' | trans %}
 {% include 'taxonomy/term/templates/partials/head' %}
 {% include 'taxonomy/term/templates/partials/entities' %}
-{% include 'taxonomy/term/templates/partials/discussion' %}
\ No newline at end of file
+{% set discussions = 'taxonomy/term/templates/partials/discussion' %}
```
Comment (inyono) [inline src/module/Ui/templates/taxonomy/term/templates/curriculum-topic-folder.twig]: missing render

File: src/module/Ui/templates/taxonomy/term/templates/curriculum-topic.twig [hunk 2]
```diff
@@ -3,4 +3,5 @@
 {% include 'taxonomy/term/templates/partials/subfolders' %}
 {% include 'taxonomy/term/templates/partials/entities' %}
 {% include 'taxonomy/term/templates/partials/groups' %}
-{% include 'taxonomy/term/templates/partials/discussion' %}
\ No newline at end of file
+{% set discussions = 'taxonomy/term/templates/partials/discussion' %}
```
Comment (inyono) [inline src/module/Ui/templates/taxonomy/term/templates/curriculum-topic.twig]: missing render

File: src/module/Ui/templates/taxonomy/term/templates/curriculum.twig [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {% set typename = 'curriculum' | trans %}
 {% include 'taxonomy/term/templates/partials/head' %}
 {% include 'taxonomy/term/templates/partials/subfolders' %}
-{% include 'taxonomy/term/templates/partials/discussion' %}
\ No newline at end of file
+{% set discussions = 'taxonomy/term/templates/partials/discussion' %}
```
Comment (inyono) [inline src/module/Ui/templates/taxonomy/term/templates/curriculum.twig]: missing render

File: src/module/Ui/templates/taxonomy/term/templates/locale.twig [hunk 4]
```diff
@@ -1,4 +1,5 @@
 {% set typename = 'locale' | trans %}
 {% include 'taxonomy/term/templates/partials/head' %}
 {% include 'taxonomy/term/templates/partials/subfolders' %}
-{% include 'taxonomy/term/templates/partials/discussion' %}
\ No newline at end of file
+{% set discussions = 'taxonomy/term/templates/partials/discussion' %}
```
Comment (inyono) [inline src/module/Ui/templates/taxonomy/term/templates/locale.twig]: missing render

File: src/module/Ui/templates/taxonomy/term/templates/topic-folder.twig [hunk 5]
```diff
@@ -1,4 +1,5 @@
 {% set typename = 'topic-folder' | trans %}
 {% include 'taxonomy/term/templates/partials/head' %}
 {% include 'taxonomy/term/templates/partials/entities' %}
-{% include 'taxonomy/term/templates/partials/discussion' %}
\ No newline at end of file
+{% set discussions = 'taxonomy/term/templates/partials/discussion' %}
```
Comment (inyono) [inline src/module/Ui/templates/taxonomy/term/templates/topic-folder.twig]: missing render

File: src/module/Ui/templates/taxonomy/term/templates/topic.twig [hunk 6]
```diff
@@ -3,4 +3,5 @@
 {% include 'taxonomy/term/templates/partials/subfolders' %}
 {% include 'taxonomy/term/templates/partials/entities' %}
 {% include 'taxonomy/term/templates/partials/groups' %}
-{% include 'taxonomy/term/templates/partials/discussion' %}
\ No newline at end of file
+{% set discussions = 'taxonomy/term/templates/partials/discussion' %}
```
Comment (inyono) [inline src/module/Ui/templates/taxonomy/term/templates/topic.twig]: missing render

**Pull request merged**